src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentControllerTest.kt

```diff
@@ -270,7 +271,7 @@ class DocumentControllerTest {
     val documentId = ::DocumentId.random()
 
     every { tokenExtractor.getTokenFromHeader(bearerToken) } returns TokenExtractor.Token(userId.toString())
-    every { recallNotificationService.generateAndStorePdf(recallId, userId, details) } returns Mono.just(Pair(documentId, "content".toByteArray()))
+    every { recallNotificationService.generateAndStorePdf(recallId, userId, details) } returns Mono.just(documentId)
```

Comment:
Wondering if we still need `generateAndStorePdf()` instead of just `generatePdf()`?
Are there any documents we generate but don't store now?

Reply:
Yes. Table of Contents. Letter to Probation. a few.